### PR TITLE
amend https://github.com/aodn/data-services/issues/488

### PR DESCRIPTION
**Pull request by lbesnard**

### CODE REVIEW

**Comment by lbesnard** [comment]
@ggalibert have a look, but this is working

File: ANMN/burst_averaged_product/burst_average.py [hunk 1]
```diff
@@ -328,7 +330,18 @@ def create_burst_average_netcdf(input_netcdf_file_path, output_dir):
...
     generate_netcdf_att(output_netcdf_obj, conf_file_generic, conf_file_point_of_truth=True)
 
-    time_burst_val_dateobj = num2date(time_burst_vals, output_netcdf_obj['TIME'].units, output_netcdf_obj['TIME'].calendar)
+    # set up original varatts for the following dim
+    varnames = ['TIME', 'LATITUDE', 'LONGITUDE', 'NOMINAL_DEPTH']
+    for varname in varnames:
+        for varatt in input_netcdf_obj[varname].__dict__.keys():
+            setattr(output_netcdf_obj[varname], varatt, getattr(input_netcdf_obj[varname], varatt))
+
+    time_comment = '%s. Time stamp corresponds to the middle of the burst measurement which lasts %s seconds.' % (getattr(input_netcdf_obj['TIME'], 'comment', ''),
```
Comment (lbesnard) [inline ANMN/burst_averaged_product/burst_average.py]: @ggalibert see mod here

**Comment by lbesnard** [comment]
[IMOS_ANMN-NRS_KOSTUZ_20151207T024417Z_NRSMAI_FV02_NRSMAI-SubSurface-24-2015-11-WQM-85-burst-averaged_END-20160507T235914Z_C-20160617T115200Z.nc.zip](https://github.com/aodn/data-services/files/319636/IMOS_ANMN-NRS_KOSTUZ_20151207T024417Z_NRSMAI_FV02_NRSMAI-SubSurface-24-2015-11-WQM-85-burst-averaged_END-20160507T235914Z_C-20160617T115200Z.nc.zip)
this is a new FV02 file

**Comment by ggalibert** [comment]
:+1:

**Pull request merged**